Gemfile

```
@@ -15,7 +15,7 @@ gem 'nokogiri', '>= 1.8.1'
 gem 'openssl'
 gem 'paper_trail', '~> 12.0'
 gem 'prawn-rails-forms', '~> 0.1.2'
-gem 'puma', '~> 4.3'
+gem 'puma'
 gem 'rails', '~> 6.1.3'
```

Might want to change this to 6.1.4 for posterity.
```suggestion
gem 'rails', '~> 6.1.4'
```